analyses/molecular-subtyping-embryonal/README.md

```diff
@@ -29,13 +29,24 @@ bash run-embryonal-subtyping.sh
 
 `00-embryonal-select-pathology-dx.Rmd` is not run via this module's shell script, as it should be run locally, tied to `release-v17-20200908`, and should not be re-rendered when there are changes to the underlying `pbta-histologies.tsv` file in future releases (see [Folder content](#folder-content) and [#748](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/748)).
 
-## Folder Content
+## Molecular subtyping HGG workflow
+
+ ![embryonal molecular subtyping workflow](./plot/embryonal_subtyping.png)
+
+
+## Molecular subtyping criterias
 
-[`00-v17-embryonal-select-pathology-dx.Rmd`](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/00-embryonal-select-pathology-dx.nb.html) is a notebook used to explore the `pathology_diagnosis` and `pathology_free_text_diagnosis` fields in the `release-v17-20200908` version of `pbta-histologies.tsv`. 
-Prior to `release-v17-20200908`, this module used `broad_histology == "Embryonal tumor"` to identify samples to be included for subtyping.
-ATRT and Medulloblastoma samples were also filtered out with this method of identifying samples.
-In future releases, the `broad_histology` values will be derived from the `pathology_diagnosis` and `molecular_subtype` values (see [#748](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/748)); this module generates the latter.
-Thus, this notebook looks to identify the samples to be included for subtyping based on the `pathology_diagnosis` and `pathology_free_text_diagnosis` values.
+First, samples was selected based on the pathology-diagnosis and pathologies_free_text-diagnosis in this [script](./00-embryonal-select-pathology-dx.R). To molecular subtype embryonal tumors, following criteria is used based on WHO cancer guidebook: 
+
+- **ETMR, C19MC-altered**: samples contain chromosome 19 amplification or the overexpression of _LIN28A_ and contain _TTYH1_ gene fusion;
+- **ETMR, NOS**: samples contain NO _TTYH1_ gene sufion and the verexpression of _LIN28A_. 
+- **CNS HGNET-MN1**: samples contain _MN1_ gene fusion.
+- **CNS NB-FOXR2**: samples contain _FOXR2_ gene fusion or the overexpression of _FOXR2_.
+- **CNS Embryonal, NOS**: Pathology diagnosis is "Neuroblastoma" and do NOT have high confidence methylation subtypes.
+- For all the other samples with Pathology diagnosis of "Neuroblastoma" and have high confidence methylation subtypes, molecular subtypes of these samples follow their methylation molecular subtypes.
```

Comment:
This isn't reflected in the image. The image contains a node for methylation classification calling "ETMR, C19MC-altered" but no other subtype options. If there are other subtypes based on methylation data I would include a reference in the README. I couldn't find this specfically in the WHO guidebook. If the subtype option is only ETMR, C19MC-altered I would somehow make the arrow extend to that node rather than have two subtype nodes classifying the same subtype.

Reply:
Here the situation is little complicated.... I think the workflow needs to read from up to bottom, left to right. I am not sure how to make it clear to viewers. But in this case, `ETMR, C19MC-altered` is first identified by chr19 amp and fusion, followed by subtyping other samples. Then for all the samples that do not have molecular subtypes, we look at the methyl classification. We only have samples with `ETMR, C19MC-altered` high confidence methylation. That's why I only put `ETMR, C19MC-altered` in the end. In my opinion, it does not conflict with what is written in README. Please let me know if that makes sense.